rosdep/python.yaml

```diff
@@ -6229,6 +6229,11 @@ python3-jsonschema:
   fedora: [python3-jsonschema]
   gentoo: [dev-python/jsonschema]
   ubuntu: [python3-jsonschema]
+python3-junitparser:
+  debian: [python3-junitparser]
+  fedora: [python3-junitparser]
+  gentoo: [dev-python/junitparser]
+  ubuntu: [python3-junitparser]
```

Comment:
@danthony06 `python3-junitparser` is available since Ubuntu Focal.

Consider `null`'ing `xenial` and `bionic`.

Reply:
Okay, done.